packages/cli/src/zosfiles/-strings-/en.ts

```diff
@@ -736,6 +736,19 @@ export default {
                      `without sequence numbers`
                 }
             },
+            SPOOL_DD: {
+                SUMMARY: "Compare content of two spool dds",
+                DESCRIPTION: "Compare content of two spool dds on your terminal (stdout).",
+                POSITIONALS: {
+                    SPOOLDDDESCRIPTION1: "The name of the job with the id of spool dd",
+                    SPOOLDDDESCRIPTION2: "The name of the job with the id of spool dd"
```

Comment:
should we differentiate them ?
ex: `the name of the first ...`  and `the name of the second ... `